thriftbp/prometheus.go

```diff
@@ -0,0 +1,53 @@
+package thriftbp
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+const (
+	serviceLabel             = "thrift_service"
+	methodLabel              = "thrift_method"
+	successLabel             = "thrift_success"
+	exceptionLabel           = "thrift_exception_type"
+	baseplateStatusLabel     = "thrift_baseplate_status"
+	baseplateStatusCodeLabel = "thrift_baseplate_status_code"
```

Comment:
There is no need to prefix labels like this. The metric is `thrift`, therefore any label naming implies thrift.

Department of redundancy department. 😄 

Although, `service` is possibly something we should reserve for outside of application naming.

Reply:
these label names are defined in the [baseplate.spec](https://github.snooguts.net/reddit/baseplate.spec/blob/master/component-apis/prom-metrics.md#thrift), so if they need to change I think they should be updated there. Thoughts?

Reply:
Yes, I think we should fix the spec. I'll send out a PR.

Reply:
Nevermind, I'm not going to argue this anymore.